Zend/tests/bug79934.phpt

```diff
@@ -0,0 +1,11 @@
+--TEST--
+Bug #79934: CRLF-only line in heredoc causes parsing error
+--FILE--
+<?php
+eval("\$s = <<<HEREDOC\r\n    a\r\n\r\n    b\r\n    HEREDOC;");
+var_dump($s);
```

Comment:
I would suggest to add variations with \n and \r as well and use something like `addcslashes(` for the output, so we see the linebreaks more explicitly.

Reply:
I updated the test to include cases with "\n", "\r" and both "\r\n" as well as a whitespace-only line.